Standards/scs-XXXX-vN-decision-record-template.md

```diff
@@ -0,0 +1,40 @@
+---
+title: _Descriptive title_
+type: Decision Record
+status: Draft
```

Comment:
should this always be `Draft`?

Reply:
Nope, I just didn't change it, since the previous Standard document also had only `Draft` there, probably to suggest that a completely new document can't be stable right from the start.

Reply:
Yes. Initially, it may only be a draft.

Reply:
This information should be reflected somewhere in the document though, no?

Reply:
These templates are meant to be used by people who have read and understood scs-0001-v1.